lib/systems/default.nix

```diff
@@ -1,9 +1,60 @@
-{ lib }:
-  let inherit (lib.attrsets) mapAttrs; in
+{ lib
+}:
+let
+ inherit (lib.attrsets) mapAttrs;
+
+ selectEmulator = platform: pkgs:
```

Comment:
Shouldn't we remove the original definition of this below?

Reply:
5dd051aa06a2